tests/analysis/Decompilation/dynamic_native/linked_list.c

```diff
@@ -0,0 +1,51 @@
+#include <malloc.h>
```

Comment:
Add to all files the copyright notice. See `tests/runtime/floating-point.c`.

Reply:
fixed